modules/vmfcore/include/vmf/jsonreader.hpp

```diff
@@ -40,13 +41,13 @@ namespace vmf
 * class JSONReader
 * \brief JSONReader class is a %IReader interface implementation for JSON format representation
 */
-class VMF_EXPORT JSONReader : public IReader
+class VMF_EXPORT JSONReader : public ReaderBase
 {
 public:
     /*!
     * \brief Default class constructor
     */
-    JSONReader();
+    JSONReader(vmf_string _compressorId = vmf_string());
```

Comment:
this will be removed because each IReader instance should detect compressor ID automatically